client/ayon_core/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -152,24 +202,147 @@ def process(self, instance):
+                        "translations and can be used to evaluate run-up that"
+                        " isn't actually translated."
+                    ),
+                }
+            },
+            "pythonPerFrameCallback": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "pythonPerFrameCallback",
+                }
+            },
+            "pythonPostJobCallback": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "pythonPostJobCallback",
+                }
+            },
+            "userAttr": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "userAttr",
+                }
+            },
+            "userAttrPrefix": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "userAttrPrefix",
+                }
+            },
+            "visibleOnly": {
+                "def": BoolDef,
+                "kwargs": {
+                    "label": "Visible Only",
+                }
+            }
+        }
+
+        defs = super(ExtractAlembic, cls).get_attribute_defs()
+
+        defs.extend([
+            UISeparatorDef("sep_alembic_options"),
+            UILabelDef("Alembic Options"),
+        ])
+
+        # The Arguments that can be modified by the Publisher
+        overrides = set(getattr(cls, "overrides", set()))
+
+        # What we have set in the Settings as defaults.
+        flags = set(getattr(cls, "flags", set()))
+
+        enabled_flags = [x for x in flags if x in overrides]
+        flags = overrides - set(override_defs.keys())
+        defs.append(
+            EnumDef(
+                "flags",
+                flags,
+                default=enabled_flags,
+                multiselection=True,
+                label="Export Flags",
+            )
+        )
+
+        for key, value in override_defs.items():
+            if key not in overrides:
+                continue
+
+            kwargs = value["kwargs"]
+            kwargs["default"] = getattr(cls, key, None)
+            defs.append(
+                value["def"](key, **value["kwargs"])
+            )
+
+        defs.append(
+            UISeparatorDef("sep_alembic_options")
+        )
+
+        return defs
+
 
 class ExtractAnimation(ExtractAlembic):
-    label = "Extract Animation"
+    label = "Extract Animation (Alembic)"
     families = ["animation"]
 
     def get_members_and_roots(self, instance):
-
         # Collect the out set nodes
         out_sets = [node for node in instance if node.endswith("out_SET")]
         if len(out_sets) != 1:
-            raise RuntimeError("Couldn't find exactly one out_SET: "
-                               "{0}".format(out_sets))
+            raise RuntimeError(
+                "Couldn't find exactly one out_SET: " "{0}".format(out_sets)
+            )
         out_set = out_sets[0]
         roots = cmds.sets(out_set, query=True)
```

Comment:
Should this have a failsafe for `cmds.sets` returning None if the set is empty as well? (Supposedly a validator catches that already, but maybe it doesn't?)